### DISCUSSION TITLE
FluidHTML?

### DISCUSSION THREAD

**Discussion by kinekt4** [downforcetech/fluidxml.php#56] [Questions]
I've been using FluidXML on a side project and I've just discovered it doesn't play well with HTML.

Looking further into, I can see that `DomDocument` calls `loadXML()` instead of `loadHTML()`

From php.net docs:

> Unlike loading XML, HTML does not have to be well-formed to load

I assume changing to `loadHTML()` will be a breaking change. Just wondering  what are my options?

**Comment by daniele-orlando** [comment]
We could extend the APIs to add a loadHtml method.
At the moment you can load the HTML with DOMDocument and then import the DOMDocument inside FluidXML.
